ckanext/dia_theme/templates/header_base.html

```diff
@@ -1,9 +1,10 @@
 {# This is copied from the CKAN source, with header_site_search moved outside .nav-collapse, and accessibility changes as per wr267900 #}
+
 {% block header_wrapper %}
     <header class="site-header" role="banner">
         {% block header_debug %}
             {% if g.debug and not g.debug_supress_header %}
-                <div class="debug">Controller : {{ c.controller }}<br/>Action : {{ c.action }}</div>
+            <div class="debug">Blueprint : {{ g.blueprint }}<br/>View : {{ g.view }}</div>
```

Comment:
The changes in this file are a merge of the latest version of the header_base template from CKAN core, and the modifications that have been made to this template for the DIA's requirements in the extension.